### DISCUSSION TITLE
Is proteus ready for FPGA boards?

### DISCUSSION THREAD

**Discussion by Trolldemorted** [proteus-core/proteus#4] [Q&A] [answered]
Hey,

Did you ever try to run proteus on FPGA boards like the Xilinx VCU-118? Is it mature enough to try that?

**Comment by martonbognar** [comment]
Hi! I personally have only synthesized Proteus for FPGAs in Xilinx Vivado but haven't run it on physical boards. We might have some collaborators though, who have loaded it onto a physical board; I will try to get more information. 

I will also upload the other files needed to synthesize the project with some instructions. Are you planning to try it out? I would be interested in how easily it goes!

**Reply by martonbognar** [reply]
I have now added some instructions for running the synthesis, let me know if this is sufficient!
https://github.com/proteus-core/proteus#synthesis

**Comment by fpiessens** [comment, marked as answer]
Hi! 
Our collaborators have put various Proteus variants on a Zynq UltraScale+ XCZU9EG FPGA using the Xilinx Vivado tools.
They also deployed several Proteus variants on the lower-end Arty A7-35T XC7A35TICSG324-1L FPGA board.

You can find more details in Chapter 4 of Thomas Van Strydonck's PhD thesis: https://lirias.kuleuven.be/3750264 

Hope this helps!
